libethereum/BlockChain.cpp

```diff
@@ -1544,7 +1544,7 @@ VerifiedBlockRef BlockChain::verifyBlock(bytesConstRef _block, std::function<voi
             try
             {
                 Transaction t(d, (_ir & ImportRequirements::TransactionSignatures) ? CheckTransaction::Everything : CheckTransaction::None);
-                m_sealEngine->verifyTransaction(_ir, t, h, 0);
+                m_sealEngine->verifyTransaction(_ir, t, h, 0); // the gasUsed vs blockGasLimit is checked later in enact function
```

Comment:
This comment isn't true anymore, is it?

Reply:
this comment is here because I spent some time trying to figure out where the actual check of the gasLimit is performed

Reply:
Ah right, we don't know how much gas will be used until we actually execute the transaction.
Ok, let this stay if you find it helpful.